examples/01_plotting/plot_surf_stat_map.py

```diff
@@ -0,0 +1,58 @@
+"""
+Demonstrate basic surface plotting with plot_surf_stat_map
+==========================================================
+
+See :ref:`plotting` for more details.
```

Comment:
> pin point reference may be :func:nilearn.plotting.plot_surf_stat_map ?

I think that this is OK, as long as there is a section about surface
visualization in the docs.